README.md

```diff
@@ -1,3 +1,95 @@
-# storey
+# Storey
 
 [![CI](https://github.com/v3io/storey/workflows/CI/badge.svg)](https://github.com/v3io/storey/actions?query=workflow%3ACI)
+
+Storey is an asynchronous flow library, for real time event processing and feature extraction.
+
+#### In This Document
+
+- [API Walkthrough](#api-walkthrough)
+- [Usage Examples](#examples)
+
+
+<a id="api-walkthrough"></a>
+## API Walkthrough
+A Storey flow consist of steps linked together by the `build_flow` function, each doing it's designated work.
```

Comment:
Probably better leave the specific function to the code example.

Reply:
Decided to leave as is for now, as it is part of the generic API and not a "Step".
@gtopper will optionally add more code examples on how to create a flow with the `.to()` API without the `build_flow` sugar